canonical/canonical-design
# Build Navigation, Header and Footer templates

## Done

- Refactor navigation to use [vanilla sliding nav](https://vanillaframework.io/docs/patterns/navigation#sliding)
- Align footer with Vanilla
- Removes some unused CSS and JS
- Adds yarn.lock to project

## QA

- Check out this branch and run locally
- Check the navigation matches that [on Vanilla](https://vanillaframework.io/docs/patterns/navigation#sliding)
- Check the dropdowns work, check the sliding nav works on small screens
- Check the footer content matches

## Issue

Fixes https://warthogs.atlassian.net/browse/WD-16707
